microindex/trailer.go

```diff
@@ -138,6 +143,13 @@ func recordToTrailer(rec *zng.Record) (*Trailer, error) {
 	if err != nil {
 		return nil, err
 	}
+	order, err := rec.AccessInt(OrderField)
+	if err != nil {
+		return nil, err
+	}
+	if order < 0 {
+		trailer.Order = zbuf.OrderDesc
+	}
```

Comment:
Nit: I think this would be a little nicer as a boolean field named `DescendingField = "descending"`.
```go
	descending, err := rec.AccessBool(DescendingField)
	if err != nil {
		return nil, err
	}
	if descending {
		trailer.Order = zbuf.OrderDesc
	}

Reply:
I like it